crates/typst/src/foundations/float.rs

```diff
@@ -65,8 +66,8 @@ impl f64 {
 
     /// Checks if a float is infinite.
     ///
-    /// For floats, there is positive and negative infinity. This function
-    /// returns `true` if the float is either positive or negative infinity.
+    /// Floats can represent positive and negative infinity. This function
+    /// returns `{true}` if the float is either an infinity.
```

Comment:
the `an` change doesn't make sense and is also incorrect grammar. leaving it as it was is fine

Reply:
I'm not a native English speaker so I may be wrong, but, from my understanding, "infinity" can be used as a noun. There are two infinities: negative infinity, and positive infinity. I can refer to one of them with "an infinity".

Reply:
It depends on the context whether "infinity" is an uncountable noun, meaning it needs no article, or a countable one. Regardless, it is wrong in this context especially with the "either" as that would have to be followed by an "or" at some point.

Reply:
Oh you're right! I meant to write "the float is an infinity" without "either". Would that be correct?

Reply:
Yeah, that'd be correct.